CMakeLists.txt

```diff
@@ -97,32 +135,57 @@ else()
     message(ERROR "libpowerloader used as neither shared or static library: not supported")
 endif()
 
-# Custom build flags
-if(MSVC)
+target_include_directories(
+    libpowerloader
+    PUBLIC $<BUILD_INTERFACE:${POWERLOADER_INCLUDE_DIR}>
+    $<INSTALL_INTERFACE:include>
+)
+
+if (MSVC)
+    target_compile_definitions(libpowerloader PUBLIC -DNOMINMAX)
     target_compile_options(libpowerloader
         PUBLIC
             /wd4275 # Disable warning C4275: We know that we use templates instances in our symbol exports.
             /wd4251 # Disable warning C4251: We know that we use templates instances in our symbol exports.
+            /permissive
```

Comment:
It should be `/permissive-` which means it have to be standard compliant, see: https://docs.microsoft.com/en-us/cpp/build/reference/permissive-standards-conformance?view=msvc-170